scripts/commit-run-scripts/commit_run.sh

```diff
@@ -17,13 +17,35 @@
 SCRIPT_PATH="$SCRIPTS_DIR/common_commit_initialization.sh"
 . $SCRIPT_PATH
 
+if [[ ! -z "${PRE_COMMIT_COMMAND}" ]]; then
+    pipe_log_info "[INFO] Pre-commit command found by path ${PRE_COMMIT_COMMAND}" "$TASK_NAME"
+    pipe_exec "docker exec ${CONTAINER_ID} ls ${PRE_COMMIT_COMMAND} > /dev/null" "$TASK_NAME"
+    if [[ $? -eq 0 ]]; then
+        pipe_log_info "[INFO] Run pre-commit command" "$TASK_NAME"
+        pipe_exec "docker exec ${CONTAINER_ID} sh -c '${PRE_COMMIT_COMMAND} ${CLEAN_UP} ${STOP_PIPELINE}'" "$TASK_NAME"
+        check_last_exit_code $? "[ERROR] There are some troubles while executing pre-commit script." \
+                        "[INFO] Pre-commit operations were successfully performed."
+    fi
+fi
+
 commit_file $FULL_NEW_IMAGE_NAME
 check_last_exit_code $? "[ERROR] Error occurred while committing temporary container" \
                         "[INFO] Temporary container was successfully committed with name: $image_name" \
                         "python $COMMON_REPO_DIR/scripts/commit_run.py ups $RUN_ID FAILURE"
 
 export tmp_container=`docker run --entrypoint "/bin/sleep" -d ${FULL_NEW_IMAGE_NAME} 1d`
 
+if [[ ! -z "${POST_COMMIT_COMMAND}" ]]; then
```

Comment:
The two code snippets differ only by the container id, can we move it to a function? We already have enough copy-pastes in this script...